src/style/icons.css

```diff
@@ -120,9 +120,51 @@
   -webkit-mask: url('./images/i_cam_tilt.svg') no-repeat center;
 }
 
-.ngm-zoom-plus-icon {
-  width: 20px;
-  height: 20px;
```

Comment:
@vladyslav-tk sorry messed this up solving the merge conflicts. The issue is that we both specified the zoom icon with different name and size. I think we should only have it once. I prefer your naming and my size what do you think?

Reply:
we can change size depending on the component. Currently, seems like we have this icon in object info and in the data catalog, according to the mockup, in both places size should be 20x20px

Reply:
ok, converted to your styling/naming